atlp-rwanda/cabal-phantom-frontend
# #CbqRAya0 Register drivers and operators

### What does this PR do?
This PR is about to register users

### Description of Task to be completed?
- Admin should register users
- Skeleton should appeared when page is loading
- Page should be responsive

### How should this be manually tested?
-  clone this repo and do git checkout ft-register-users-CbqRAya0
- Open login page with this [link](https://cabal-fronte-ft-registe-x0qfcc.herokuapp.com/login)
- Login as admin with credentials below
       {
              email: admin@phantom.com
              password: admin
      }
- Click on **users**, then click on **Add** button (on the right side) to add new user
- Fill all fields and check your email after register successful
 
### **Relevant pivots tracker**

- [#CbqRAya0](https://trello.com/c/CbqRAya0/36-register-drivers-operators)

### Screenshot
<img width="1440" alt="Screen Shot 2020-12-17 at 11 34 40" src="https://user-images.githubusercontent.com/68019992/102469832-e39f4680-405b-11eb-9897-7e19caf774fc.png">
